calico-cloud/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Calico Cloud
+description: Learn about Calico Cloud.
+canonical_url: '/calico-cloud/index'
+redirect_from: '/calico-enterprise'
+---
+
+![calico-cloud]({{site.baseurl}}/images/calico-cloud-small.png)
+
+## You can solve these 3 problems today...with Calico Cloud
+
+- **Workload access controls**
+
+    Implement pod-level workload access controls and protect containerized environments in the Kubernetes cluster from outside threats, while enabling applications and workloads to securely communicate with resources outside the cluster behind a firewall or other control point.
+
+- **Compliance**
+
+    Create compliance policies for any compliance framework including PCI, SOC 2, and more. Continuously monitor compliance for your container, Kubernetes, and cloud environment. Retain a daily history of your compliance status. Generate on-demand or scheduled audit reports.
+
+- **Faster troubleshooting**
+
+    Enable faster troubleshooting of Kubernetes workloads and applications with Service Graph, Packet Capture, anomaly detection, and performance hotspots, leading to shorter time-to-resolution, less application downtime, and improved quality of service.
+
+#### Don't have a Calico cluster? Easy...
+
+{% include open-new-window.html text='Try it now' url='https://www.tigera.io/tigera-products/cloud-trial' %}
+
+#### Already have a Calico cluster? You are **5 minutes** away from connecting to Calico Cloud
+
+1. Identify the cluster you want to use and apply the following manifest.
+
+   You can use any Calico operator-installed cluster that is configure with `kubectl`.
+
+   ```bash
+   kubectl apply -f https://storage.googleapis.com/dev-calicocloud-installer/manifests/cc-operator/latest/deploy.yaml
+   ```
+
+1. Add your email address to get an invite and a license.
+
+   ```bash
+   kubectl apply -f https://storage.googleapis.com/dev-calicocloud-installer/manifests/cc-operator/latest/deploy.yaml
+   ```
+
+1. Accept your invite to join {{site.prodname}}.
+
+    Open the email, fill in the brief onboarding dialog, and in about 5 minutes, your cluster will appear in the Calico Cloud Manager UI drop-down menu. That's it! You're ready to go!
+
```

Comment:
It might be worth adding the following command so they can check the current status of the installer:
```
kubectl get installers.operator.calicocloud.io -n calico-cloud default -o jsonpath='{.status.state}'
```

Reply:
Good idea! Added.